### PR TITLE
Serve PAC file from paxy executable

### PR DESCRIPTION
**Pull request by luke-clifton**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -12,6 +13,20 @@ import (
...
 
+type Server struct {
+	Proxy  *proxy.Proxy
+	Source []byte
+}
+
+func (server *Server) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	if r.Method == http.MethodGet && r.URL.Host == "" && r.URL.Path == "/pac" {
+		w.Write([]byte(server.Source))
+		w.Write([]byte("\n"))
```
Comment (robbiemcmichael) [inline main.go:24]: What's the extra newline character for? Shouldn't we reproduce exactly the same bytes that we read from the file?

**Comment by robbiemcmichael** [comment]
Did you want to add something to the readme like you did in the last PR about it serving the PAC file file from the `/pac` path?

File: main.go (hunk 1 above)
Comment (luke-clifton) [reply main.go]: Hmm, for some reason I thought it wasn't sending the last line without it, but I double checked this morning and it works fine, so will remove.

**Review by robbiemcmichael — APPROVED** [review]
Thanks, looks good.

**Pull request merged**